tools/ai-evals/azsdk-mcp/azsdk-mcp.csproj

```diff
@@ -0,0 +1,31 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <RootNamespace>azsdk_mcp_evals</RootNamespace>
```

Comment:
The naming convention we tend to use would be something like `Azure.Sdk.Tools.McpEvals`, for both the root namespace, any child namespaces, and the csproj filename.